src/App.css

```diff
@@ -1,4 +1,334 @@
-body {
+:root {
+  --blue: #0290ff;
+  --black: #121212;
+  --grey: grey;
+  --pale-grey: #f5f6fa;
+  --genre-size: 0.875rem;
+  --book-title-size: 1.375rem;
+  --logo-size: 1.875rem;
+  --nav-link-size: 0.813rem;
+  --form-heading-size: 1.25rem;
+  --secondary-btn-color: #4386bf;
+  --robotoSlab: "Roboto Slab", serif;
+  --montserrat: "Montserrat", sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+  padding: 0;
   margin: 0;
-  background-color: beige;
+
+  /* border: 1px solid blue; */
+}
+
+body {
+  background-color: var(--pale-grey);
+}
+
+button,
+a {
+  cursor: pointer;
+}
+
+header {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 20px;
+  background: white;
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  font-family: var(--montserrat);
+  font-size: var(--logo-size);
+  font-weight: bold;
+  color: var(--blue);
+}
+
+nav {
+  display: flex;
+  gap: 10px;
+}
+
+nav > a {
+  font-family: var(--montserrat);
+  font-size: var(--nav-link-size);
+  color: var(--black-two);
+  text-decoration: none;
+  padding: 10px;
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+.books-list {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  padding: 20px;
+}
+
+/* book styling */
+.book-card {
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+  padding: 2rem;
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+.book-info {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+.category {
+  opacity: 0.5;
+  font-family: var(--montserrat);
+  font-size: 0.875rem;
+  font-weight: bold;
+  color: var(--black);
+}
+
+.book-title {
+  font-family: var(--robotoSlab);
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: var(--black-two);
+}
+
+.book-author {
+  font-family: var(--robotoSlab);
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--secondary-btn-color);
+}
```

Comment:
- [ ] `required`
- [ ] Please put the `book author`, `Book title` `category` and the `add` button in the same line just as the template provided.
- [ ] Please see screenshots below

`your template`
![image](https://github.com/jash-debug/bookstore/assets/66230306/4de2d996-ae56-446e-a7e6-5577221cc3e3)

`original template`
![image](https://github.com/jash-debug/bookstore/assets/66230306/c731ed79-145f-4895-9ea1-6bcf004f25c5)